src/app/app.component.scss

```diff
@@ -4,8 +4,8 @@
 .navbar-header {
     span{
 		color: $color-pf-white;
-		padding: $pad5 $pad10;
-		font-size: $font18;
+		padding: em(5) em(10);
+		font-size: em(18);
```

Comment:
So is this 18 times the base pixel definition? Normally, it would be `font-size: 1.5em` or something along those lines.